interfaces/Portalicious/src/app/pages/project-registrations/components/export-registrations/export-registrations.component.html

```diff
@@ -26,35 +26,6 @@
     [exportType]="ExportType.allRegistrations"
   />
 </app-confirmation-dialog>
-<app-confirmation-dialog
-  #exportDuplicatesDialog
```

Comment:
This `exportDuplicatesDialog` needs to be removed from the TS file too :) (Shame that eslint doesn't complain about this....)